### PR TITLE
[8.0] update travis config

### PR DESCRIPTION
**Pull request by gurneyalex**
use latest standards

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
 
 virtualenv:
   system_site_packages: true
 
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (pedrobaeza) [inline .travis.yml:24]: You can add `--depth=1` (although it's not very heavy)

**Comment by StefanRijnhart** [comment]
:+1: even without the depth parameter. @gurneyalex is a busy man ;-)

**Comment by pedrobaeza** [comment]
Also a rebase is needed.

**Comment by StefanRijnhart** [comment]
Ah, sorry I missed that.

**Comment by yvaucher** [comment]
Superseeded by https://github.com/OCA/stock-logistics-barcode/pull/19

**Pull request closed without merging**